### ISSUE TITLE
Expected input function to AppsyncResolver to be compiled by Functionless

### ISSUE THREAD

**Issue by volkanunsal** [functionless/functionless#220]
I'm getting this error in my Jest suite on a fresh project:

```
Expected input function to AppsyncResolver to be compiled by Functionless. Make sure you have the Functionless compiler plugin configured correctly.
```

Could be related to #190. Anyone know how to solve it?

**Comment by thantos** [comment]
Try running 

`npx ts-patch check`

if none of the rows show installed, try running 

`npx ts-patch install -s`

and try again.

This error generally means that the Functionless compiler which should be configured in the `tsconfig.json` never ran.

**Comment by sam-goodwin** [comment]
How did you set up your project? 

Follow these steps? https://functionless.org/docs/getting-started#add-to-an-existing-cdk-project

**Comment by sam-goodwin** [comment]
@thantos the error should link to the docs for that error code.

**Comment by volkanunsal** [comment]
@sam-goodwin I used `projen` generator.

**Comment by volkanunsal** [comment]
@thantos I ran `npx ts-patch install -s` twice. `npx ts-patch check` shows me the following:

```
Checking TypeScript v4.7.3 installation in /Users/newuser/w/geo/geotable-backend/node_modules/typescript

[+] tsc.js is patched with ts-patch version 2.0.1
[-] tsserverlibrary.js is not patched.
[+] typescript.js is patched with ts-patch version 2.0.1
[-] typescriptServices.js is not patched.
[-] tsserver.js is not patched.
```

**Comment by sam-goodwin** [comment]
> @sam-goodwin I used `projen` generator.

What does your .projenrc.js look like?

When did you get the error? When running the cli or from vs code?

**Comment by sam-goodwin** [comment]
@thantos i think I found the problem. Plug-ins aren't added to tsconfig.dev.json

https://github.com/functionless/functionless-projen/blob/main/src/component.ts

**Comment by volkanunsal** [comment]
@sam-goodwin Yeah, that fixes it for me. I'll try to modify projen file to add them. I'm new to projen. I fyou have a patch I can paste in quickly let me know...

**Comment by sam-goodwin** [comment]
We will patch functionless-projen and release a new version within the hour. 

See https://github.com/functionless/functionless-projen/issues/5

**Comment by sam-goodwin** [comment]
> @sam-goodwin Yeah, that fixes it for me. I'll try to modify projen file to add them. I'm new to projen. If you have a patch I can paste in quickly let me know...

Projen is CDK Constructs for your project config. There is a field called `tsconfigDev` on the project where you can set properties within .projenrc.js. 

```ts
project.tsconfigDev.compilerOptions.plugins = [{
  transform: "functionless/lib/compiler"
}] 
```

**Comment by sam-goodwin** [comment]
@volkanunsal thanks for reporting this bug. Fix is out in v0.0.5 of `@functionless/projen`.

* https://github.com/functionless/functionless-projen/releases/tag/v0.0.5
* https://www.npmjs.com/package/@functionless/projen

**Comment by sam-goodwin** [comment]
@volkanunsal just a heads up. Functionless is a pre-release project being built out in the open - there are still bugs and limitations for us to work through.
